Yotpo/Yotpo/composer.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "yotpo/module-yotpo",
```

Comment:
I think we can use a composer repo.
about the name i'm not sure if it's required- but it sure is a convention..it's like that in all the modules i saw.